src/screens/TopicMemberScreen/index.js

```diff
@@ -16,7 +9,6 @@ import dimen from '../../utils/dimen';
 import {getAllMemberTopic} from '../../service/topics';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import {colors} from '../../utils/colors';
-import Search from '../DiscoveryScreenV2/elements/Search';
 import StringConstant from '../../utils/string/StringConstant';
 import UsersFragment from '../DiscoveryScreenV2/fragment/UsersFragment';
 import {Context} from '../../context';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of unused imports such as `Search` and `withInteractionsManaged` is a good practice for maintaining clean and readable code.
```diff
- import Search from '../DiscoveryScreenV2/elements/Search';
- import {withInteractionsManaged} from '../../components/WithInteractionManaged';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->